dspace-api/src/main/java/org/dspace/discovery/SolrServiceImpl.java

```diff
@@ -1966,17 +1992,11 @@ protected DSpaceObject findDSpaceObject(Context context, SolrDocument doc) throw
         }
         HttpHost hostURL = (HttpHost)(getSolr().getHttpClient().getParams().getParameter(ClientPNames.DEFAULT_HOST));
 
-        HttpGet method = new HttpGet(hostURL.toHostString() + "");
-        try
-        {
-            URI uri = new URIBuilder(method.getURI()).addParameter("q",query.toString()).build();
-        }
-        catch (URISyntaxException e)
-        {
-            throw new SearchServiceException(e);
-        }
+        HttpPost post = new HttpPost(hostURL.toHostString());
+        List<NameValuePair> postParameters = new ArrayList<>();
+        postParameters.add(new BasicNameValuePair("q",query.toString()));
```

Comment:
I don't see where postParameters is ever attached to the request.

Reply:
Thanks @mwoodiupui , good catch. I've fixed that bug now, see: 3d2b5bc